platform-tests/src/acceptance/cf_ssh_test.go

```diff
@@ -24,4 +37,41 @@ var _ = Describe("CF SSH", func() {
 		Expect(cfSSH).To(Exit(0))
 		Expect(cfSSH).To(Say("load average:"))
 	})
+
+	It("allows uploading a large payload via standard ssh client", func() {
+		// FIXME: Increase to 10GB once the following issue is solved:
+		// https://github.com/cloudfoundry/cli/issues/1098
+		const payload = 1*GIGABYTE + 900*MEGABYTE
+		timeout := 600
+		appName := generator.PrefixedRandomName("CATS-APP-")
+		Expect(cf.Cf(
+			"push", appName,
+			"-b", config.StaticFileBuildpackName,
+			"-p", "../../example-apps/static-app",
+			"-d", config.AppsDomain,
+			"-i", "1",
+			"-m", "64M",
+		).Wait(CF_PUSH_TIMEOUT)).To(Exit(0))
+
+		cfSSHCommand := exec.Command("/usr/bin/cf", "ssh", appName, "-c", "cat > /dev/null")
+		sshStdin, err := cfSSHCommand.StdinPipe()
+		Expect(err).NotTo(HaveOccurred())
+
+		file, err := os.Open("/dev/zero")
+		Expect(err).NotTo(HaveOccurred())
+
+		go func() {
+			defer sshStdin.Close()
+			defer GinkgoRecover()
+			copied, error := io.CopyN(sshStdin, file, payload)
```

Comment:
`error` is a built-in type, so we should probably avoid using a variable with the same name as that can get confusing.